vue/sbc-common-components/src/services/keycloak.services.ts

```diff
@@ -116,6 +116,9 @@ class KeyCloakService {
               resolve()
             }
             redirectUrl = redirectUrl || `${window.location.origin}${process.env.VUE_APP_PATH}`
+            if (logoutSiteMinder) {
+              redirectUrl = `https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?returl=${redirectUrl}&retnow=1`
```

Comment:
What will be the URL for prod siteminder?

Reply:
That is the prod URL. I don't think the TEST one works correctly.